package.json

```diff
@@ -108,8 +110,8 @@
     "sw-precache-webpack-plugin": "^0.11.5",
     "uglifyjs-webpack-plugin": "^1.2.5",
     "uuid": "^3.2.1",
-    "webpack": "^4.8.3",
-    "webpack-bundle-analyzer": "^2.12.0",
+    "webpack": "3.10.0",
```

Comment:
We're downgrading our Webpack? That seems off...

Reply:
Forgot to mention this. Adding explainer to the main description.